stackable-bcrypt/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+use std::process::exit;
+
+#[derive(Parser)]
+#[clap(
+    about = "bcrypt commandline tool",
+    long_about = "A tiny command line tool to hash a given string using the bcrypt algorithm.\
+        Default cost used is 10, but can be configured. \
+        The output is only the hash, so that this can be used for automation purposes without any parsing.",
+    author = "Stackable GmbH - info@stackable.de"
```

Comment:
```suggestion
    author
```

Clap can automagically pull this in from Cargo.toml.